qt/python/mantidqt/utils/qt/__init__.py

```diff
@@ -130,7 +131,12 @@ def create_action(parent, text, on_triggered=None, shortcut=None,
     if on_triggered is not None:
         action.triggered.connect(on_triggered)
     if shortcut is not None:
-        action.setShortcut(shortcut)
+        if isinstance(shortcut, list):
```

Comment:
Could we consider using `collections.Iterable` instead of `list`? That way we can support tuple too.

Reply:
Apparently, strings are instances of iterable too, so this change breaks the single shortcuts. I can either go for `not isinstance(shortcut, str)` or have both `isinstance(shortcut, collections.Iterable) and not isinstance(shortcut, str)`

```python
In [3]: a = "a"

In [4]: isinstance(a, collections.Iterable)
Out[4]: True
```

Reply:
This is fair - they are the two things that are worth covering.